coalib/output/ConsoleInteraction.py

```diff
@@ -154,6 +154,8 @@ def acquire_actions_and_apply(console_printer,
     :param result:          A derivative of Result.
     :param file_dict:       A dictionary containing all files with filename as
                             key.
+    :param apply_single:    The action that should be applied for all results.
+                            False if it's not used.
```

Comment:
The commit shortlog still stinks. Try:
Apply_single: Add documentation for param